CMSgov/onemac
# OY2 9722 - Package view of the Submission dashboard

Story: https://qmacbis.atlassian.net/browse/OY2-9722
Endpoint: https://d2df36a1xmlwr9.cloudfront.net/dashboard

### Details

Migrating the user interface to a package view is a multi-step process.  In this first iteration, a new data model is implemented to manage all the new functionality.  However, in order to keep a viable application while parting out the features into manageable chunks, we are running both the old and new data models simultaneously, using each one where we can best maintain functionality.

### Changes
- Cleaned up the ID checking in the submission functions
- Added the one table
- updated submit functionality to submit to both submissions table and one table
- created the PackageList which is the Package Version of the Submissions List
- converted package-exists to now use the one table.

### Implementation Notes


### Test Plan

1. Use statesubmitteractive@cms.hhs.local and submit every type of form (and all three waiver action types), using proper ids and state codes, as needed.
2. verify all submissions correctly display on the submission list "dashboard"
3. change url to /packagelist
4. Verify only the package types are shown, the status should reflect the last action taken.
  a. Be aware it may take a bit for the package view to update.
5. click the package id - should be taken to the submission details (the package details are covered in another story)
